dist/common/scripts/scylla_setup

```diff
@@ -107,8 +107,14 @@ def interactive_choose_swap_size():
         except Exception as e:
             print(f'"{s}" is not valid number, input again')
 
+pkg_files = {
+        'scylla-jmx': scylladir_p() / 'jmx/scylla-jmx',
+        'scylla-tools': scylladir_p() / 'share/cassandra/bin/cqlsh'
+        }
```

Comment:
Don't this need to be adjusted for $PRODUCT? I suppose it can be fixed later since the problem already exists.